Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -124,15 +144,18 @@
                              reducedRecHits*
                              cosmicDCTracksSeq
                              )
-
 # XeXe data with pp reco
-from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
-from RecoHI.HiCentralityAlgos.HiCentrality_cfi import hiCentrality
-from RecoHI.HiCentralityAlgos.HiClusterCompatibility_cfi import hiClusterCompatibility
-_highlevelreco_HI = highlevelreco.copy()
-_highlevelreco_HI += hiCentrality
-_highlevelreco_HI += hiClusterCompatibility
-pp_on_XeXe_2017.toReplaceWith(highlevelreco, _highlevelreco_HI)
+#from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
```

Comment:
Why did you comment out this part?

Reply:
This part was in the head, but not in @Dr15Jones 's commit.  I commented it out to avoid potential conflicts.  Do we need these back on?

Reply:
This has nothing to do with fastsim, and it is needed for setting up the pp_on_XeXe_2017 era
Please put it back